src/shared/components/post/post.tsx

```diff
@@ -909,9 +909,8 @@ export class Post extends Component<any, PostState> {
   }
 
   async handleTransferCommunity(form: TransferCommunity) {
-    const transferCommunityRes = await HttpService.client.transferCommunity(
-      form,
-    );
+    const transferCommunityRes =
+      await HttpService.client.transferCommunity(form);
```

Comment:
Its telling you the error is in prettier here: https://woodpecker.join-lemmy.org/repos/135/pipeline/1789/5#L6

You may need to run `prettier --check "src/**/*.{ts,tsx,js,css,scss}"`

If you've set up the repo correctly, it should have automatically run lint-staged on any commit, so something's going on there.

EDIT: It might be an issue with yarn 2+, so I'll make a PR to fix that with husky.

Reply:
You'll need to do this again, prettier is still failing.

Reply:
```
yarn prettier --check "src/**/*.{ts,tsx,js,css,scss}"
yarn run v1.22.19
$ /home/user/src/lemmy-ui/node_modules/.bin/prettier --check 'src/**/*.{ts,tsx,js,css,scss}'
Checking formatting...
All matched files use Prettier code style!
Done in 11.09s
```

Don't know why.